openquake/hazardlib/contexts.py

```diff
@@ -837,7 +837,9 @@ def get_mean_stds(self, ctxs):
             recarrays = [self.recarray(ctxs)]
         if any(hasattr(gsim, 'gmpe_table') for gsim in self.gsims):
             assert len(recarrays) == 1, len(recarrays)
-            recarrays = split_array(recarrays[0], U32(recarrays[0].mag*100))
+            recarrays = split_array(
+                recarrays[0], numpy.round(recarrays[0].mag * 100).astype(int)
+            )
```

Comment:
Here it really must be an uint32 otherwise numba will complain. Please do a `pip install numba` and check.

Reply:
Apparently, the `venv` we set, already had `numba` and it still worked.

![test](https://user-images.githubusercontent.com/7849113/166894872-eb92c6ac-9f99-4f7c-9c1d-30e2c3ed558a.PNG)

Should I still cast `U32()` on the second argument?

Reply:
I would keep the U32 as it was before, without round, to make sure that the behavior is the same as before.

Reply:
You are keeping the round here! I mean going back to `U32(recarrays[0].mag*100)` ;-)

Reply:
Ah! My bad haha
But in that case, users must use magnitudes in max 2dp, will these be okay?

Reply:
It is already using 2 decimal points and was approved by @mmpagani

Reply:
Gotcha, they are back online.